.eslintrc

```diff
@@ -0,0 +1,53 @@
+{
+  "ecmaFeatures": {
+    "jsx": true,
+    "modules": true
+  },
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true
+  },
+  "parser": "babel-eslint",
+  "rules": {
+    "eol-last": 1,
+    "no-unused-vars": [2, {"varsIgnorePattern": "^React$"}],
+    "max-len": [1, 120, 4],
```

Comment:
These rules should be verbatim what we have in blueocean-plugin and dashboard right now. 120 width seems reasonable to me since it's not 1995 and we're not on 15" CRT's ;)